src/EnergyPlus/SurfaceGeometry.hh

```diff
@@ -81,6 +82,12 @@ namespace SurfaceGeometry {
     extern int const UnreconciledZoneSurface; // interim value between entering surfaces ("Surface") and reconciling
     // surface names in other zones
 
+    enum enclosureType
+    {
+        RadiantEnclosures,
+        SolarEnclosures
+    };
```

Comment:
Yay! Enums!  We can in the future consider going to more `enum class`, but enums are definitely 10000% better than regular `int const` values.